pkgs/by-name/pi/pint/package.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+, testers
+, pint
+}:
+
+buildGoModule rec {
+  pname = "pint";
+  version = "0.51.0";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = "pint";
+    rev = "v${version}";
+    hash = "sha256-d3KZPPeQJBqdrr81YLusYHc5jLChC1Rf5SYeP/QMeo8=";
+  };
+
+  vendorHash = "sha256-9tJL33Qtu0xAmRnAP0BwM6CIfGs+GEG864e89XrpfLM=";
+
+  subPackages = [ "cmd/pint" ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.version=${version}"
+    "-X=main.commit=${src.rev}"
+  ];
+
+  doCheck = false;
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion {
+      inherit version;
```

Comment:
This function already sources the version to test for from the package

https://github.com/NixOS/nixpkgs/blob/366b54b81097701b05276d1b9b63622c558abe17/pkgs/build-support/testers/default.nix#L59-L61

```suggestion
```